examples/tutorial/jupyter/execution/pandas_on_dask/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM continuumio/miniconda3
+
+COPY ./requirements.txt ./requirements.txt
+
+RUN conda install -c conda-forge psutil setproctitle
```

Comment:
See the related comment.

Reply:
Answered here https://github.com/modin-project/modin/pull/4139#discussion_r813958147.